src/rotator_library/providers/anthropic_provider.py

```
@@ -0,0 +1,948 @@
+
+                            if current_event:
+                                for chunk in self._anthropic_event_to_openai_chunks(
+                                    current_event, data, model, stream_state
+                                ):
+                                    yield litellm.ModelResponse(**chunk)
+
+            except httpx.HTTPStatusError:
+                raise
+            except Exception as e:
+                file_logger.log_error(f"Error during Anthropic stream: {e}")
+                lib_logger.error(f"Anthropic stream error: {e}", exc_info=True)
```

Ruff (0.15.4)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)